elastic/tests/test_elastic.py

```diff
@@ -45,63 +46,76 @@ def test__get_urls():
     instance = {'url': URL}
     elastic_check = ESCheck('elastic', {}, instances=[instance])
 
-    health_url, stats_url, pshard_stats_url, pending_tasks_url = elastic_check._get_urls([])
+    health_url, stats_url, pshard_stats_url, pending_tasks_url, slm_url = elastic_check._get_urls([])
     assert health_url == '/_cluster/health'
     assert stats_url == '/_cluster/nodes/_local/stats?all=true'
     assert pshard_stats_url == '/_stats'
     assert pending_tasks_url is None
+    assert slm_url is None
 
-    health_url, stats_url, pshard_stats_url, pending_tasks_url = elastic_check._get_urls([1, 0, 0])
+    health_url, stats_url, pshard_stats_url, pending_tasks_url, slm_url = elastic_check._get_urls([1, 0, 0])
     assert health_url == '/_cluster/health'
     assert stats_url == '/_nodes/_local/stats?all=true'
     assert pshard_stats_url == '/_stats'
     assert pending_tasks_url == '/_cluster/pending_tasks'
+    assert slm_url is None
 
-    health_url, stats_url, pshard_stats_url, pending_tasks_url = elastic_check._get_urls([6, 0, 0])
+    health_url, stats_url, pshard_stats_url, pending_tasks_url, slm_url = elastic_check._get_urls([6, 0, 0])
     assert health_url == '/_cluster/health'
     assert stats_url == '/_nodes/_local/stats'
     assert pshard_stats_url == '/_stats'
     assert pending_tasks_url == '/_cluster/pending_tasks'
+    assert slm_url is None
 
```

Comment:
Are you missing a test case where `instance["cluster_stats"] = False` and version >=`[7, 4, 0]` for `slm_url`? Or is `slm_url` only relevant for cluster stats?